class PassportSevaRunner {

     public static void main(String seva[]){
	    
		boolean userIsRegistered =   PassportSeva.registerUser("Baba","Das","abcd","abcd");
	     System.out.println("Is USer Registered "+ userIsRegistered);
		 
		 
		      PassportSeva.getUserInfo();
	}


}